index.js

```diff
@@ -0,0 +1,293 @@
+'use strict'
+
+require('make-promises-safe')
+
+const _ = require('lodash')
```

Comment:
I would not use _ if possible, not a fan.

Reply:
Not a fan of lodash as a library or the character "_"?